packages/api/src/beacon/server/beacon.ts

```diff
@@ -23,7 +23,7 @@ export function getRoutes(config: ChainForkConfig, api: ServerApi<Api>): ServerR
         const response = await api.getBlock(...reqSerializers.getBlock.parseReq(req));
         if (response instanceof Uint8Array) {
           // Fastify 3.x.x will automatically add header `Content-Type: application/octet-stream` if Buffer
-          return Buffer.from(response);
+          return Buffer.from(response.buffer, response.byteOffset, response.byteLength);
```

Comment:
Fastify does this same [logic](https://github.com/fastify/fastify/blob/13f9b6eb9083e5f5dcd8de5895adc81101603db4/lib/reply.js#L186) for Uint8Array, so might consider just passing as is for simplicity.